src/lwc/geBatchGiftEntryTable/geBatchGiftEntryTable.js

```diff
@@ -101,7 +105,8 @@ export default class GeBatchGiftEntryTable extends LightningElement {
 
     @api
     handleSectionsRetrieved(sections) {
-        this.initColumns(this.buildColumns(sections));
+        let columns = this.buildColumns((sections));
```

Comment:
Really minor - 
```suggestion
        const columns = this.buildColumns((sections));
```

Reply:
@vmartinov-sforg with the resolution of the larger merge conflict, this should be resolved